build/ci/evergreen.yml

```diff
@@ -1199,17 +1199,37 @@ tasks:
           MCLI_PRIVATE_API_KEY: ${cloud_manager_private_api_key}
           MCLI_PUBLIC_API_KEY: ${cloud_manager_public_api_key}
           MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
-          GCP_CREDENTIALS: ${logs_decrypt_gcp_credentials}
-          AWS_ACCESS_KEY: ${logs_decrypt_aws_access_key}
-          AWS_SECRET_ACCESS_KEY: ${logs_decrypt_aws_secret_access_key}
           LOCAL_KEY: ${logs_decrypt_local_key}
           KMIP_CA: ${logs_decrypt_kmip_ca}
           KMIP_CERT: ${logs_decrypt_kmip_cert}
+          MCLI_SERVICE: cloud-manager
+          E2E_TAGS: cloudmanager,decrypt
+  - name: atlas_decrypt_e2e
+    tags: [ "e2e","generic" ]
```

Comment:
[q] does this test talk to AWS,GCP and AZURE? how generic is it? should it be moved to it's own target, it's it ok to run on every PR cost wise?